test/cli-validator/tests/expectedOutput.js

```diff
@@ -91,13 +91,13 @@ describe('cli tool - test expected output - Swagger 2', function() {
     expect(capturedText[8].match(/\S+/g)[2]).toEqual('54');
     expect(capturedText[12].match(/\S+/g)[2]).toEqual('59');
     expect(capturedText[16].match(/\S+/g)[2]).toEqual('108');
-    expect(capturedText[21].match(/\S+/g)[2]).toEqual('36');
-    expect(capturedText[25].match(/\S+/g)[2]).toEqual('59');
-    expect(capturedText[29].match(/\S+/g)[2]).toEqual('197');
-    expect(capturedText[33].match(/\S+/g)[2]).toEqual('108');
-    expect(capturedText[37].match(/\S+/g)[2]).toEqual('131');
-    expect(capturedText[41].match(/\S+/g)[2]).toEqual('134');
-    expect(capturedText[45].match(/\S+/g)[2]).toEqual('172');
+    //expect(capturedText[21].match(/\S+/g)[2]).toEqual('36');
```

Comment:
Rather than commenting this out, we should change it to assert for the validation that was moved from "warnings" to "errors" by this change

```suggestion
    expect(capturedText[21].match(/\S+/g)[2]).toEqual('172');
```